tests/functional/s/string/string_formatting.py

```diff
@@ -65,7 +65,7 @@ def print_good():
 
 def pprint_bad():
     """Test string format """
-    "{{}}".format(1) # [too-many-format-args]
+    "{{}}".format(1) # [format-string-without-interpolation]

```

Comment:
```suggestion
    "{{}}".format(1) # [too-many-format-args]

```
Doing it this way is a functional test for the old_names we added :)